<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSelectPlots</class>
 <widget class="QDialog" name="DSelectPlots">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select plots</string>
  </property>
  <widget class="QLabel" name="textLabel7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The plot files can be selected individually by pressing &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;Ctrl&lt;/span&gt; and clicking on each plot file. Alternatively, you can use a group selection by pressing &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;Shift&lt;/span&gt; and clicking on the first and last plot file.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>390</y>
     <width>176</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btSelAll">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>110</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Select all</string>
    </property>
    <property name="shortcut">
     <string>Alt+S</string>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Plot files available:</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btUnselAll">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>110</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Unselect all</string>
    </property>
    <property name="shortcut">
     <string>Alt+U</string>
    </property>
   </widget>
   <widget class="QListWidget" name="plotFiles">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>271</width>
      <height>231</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btSelAll</tabstop>
  <tabstop>btUnselAll</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btSelAll</sender>
   <signal>clicked()</signal>
   <receiver>DSelectPlots</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btUnselAll</sender>
   <signal>clicked()</signal>
   <receiver>DSelectPlots</receiver>
   <slot>unselectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DSelectPlots</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DSelectPlots</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectAll()</slot>
  <slot>unselectAll()</slot>
 </slots>
</ui>
